core/client/app/initializers/trailing-history.js

```diff
@@ -1,5 +1,4 @@
 import Ember from 'ember';
-/*global Ember */
```

Comment:
This keeps throwing me linter errors - so I removed it - I think it is a throwback?